pjsip/src/pjsua-lib/pjsua_aud.c

```diff
@@ -154,6 +154,15 @@ PJ_DEF(pj_status_t) pjsua_call_get_stream_info( pjsua_call_id call_id,
     }
 
     call_med = &call->media[med_idx];
+
+    if (call_med->state == PJSUA_CALL_MEDIA_NONE ||
+        call_med->state == PJSUA_CALL_MEDIA_ERROR ||
+        call_med->dir == PJMEDIA_DIR_NONE)
```

Comment:
For double hold scenario, the media dir will be NONE, so we should remove this condition.

Alternatively, if you want to be safe, just check directly if the stream is not NULL, i.e.:
```
if ((call_med->type == PJMEDIA_TYPE_AUDIO && !call_med->strm.a.stream) ||
     (call_med->type == PJMEDIA_TYPE_VIDEO && !call_med->strm.v.stream))
```